### PR TITLE
committing iot map

### PR DESCRIPTION
**Pull request by ssdatar**
Map assets in `iot-logistics/time-map/`

### CODE REVIEW

File: iot-logistics/time-map/index.html [hunk 1]
```diff
@@ -0,0 +1,153 @@
...
+    </style>
+
+    <!-- UPDATE VERSION # IF YOU NEED TO REPUBLISH -->
+    <link href='http://fonts.googleapis.com/css?family=Lato&subset=latin,latin-ext' rel='stylesheet' type='text/css' />
```
Comment (aturley) [inline iot-logistics/time-map/index.html]: Please change this from `http` to `https`.

File: iot-logistics/time-map/index.html [hunk 2]
```diff
@@ -0,0 +1,153 @@
...
+
+    <!--
+    ============================
+    *** DO NOT GO BELOW HERE ***
+    ============================
+    -->
+
+
+    <!-- (begin) globe iframe embed -->
+    <script src='https://apps.bostonglobe.com/common/js/pym/pym-1.3.2.min.js'></script>
```
Comment (aturley) [inline iot-logistics/time-map/index.html]: Is it going to be a problem to pull this from the Boston Globe site rather than hosting it ourselves?

**Comment by aturley** [comment]
Please remove the `.DS_store` files.

File: iot-logistics/time-map/index.html (hunk 2 above)
Comment (ssdatar) [reply iot-logistics/time-map/index.html]: This is not needed. I just updated my commit

File: iot-logistics/time-map/index.html (hunk 1 above)
Comment (ssdatar) [reply iot-logistics/time-map/index.html]: Fixed. Sorry about this! Didn't realize Google's default link was http

**Pull request merged**